cmd/boulder-va/gsb.go

```diff
@@ -78,19 +88,43 @@ func (sb gsbAdapter) IsListed(url string) (string, error) {
 	return "", nil
 }
 
+// gsbLogAdapter adapts a blog.Logger to the io.Writer interface used by the
+// Google safebrowsing client for a logger. All messages written to the Writer
+// by the library will be adapter to the logger's Info method.
+type gsbLogAdapter struct {
+	log blog.Logger
+}
+
+func (a gsbLogAdapter) Write(b []byte) (int, error) {
+	a.log.Info(string(b))
+	return len(b), nil
+}
+
 // newGoogleSafeBrowsingV4 constructs a va.SafeBrowsing instance using the new
 // Google upstream Safe Browsing version 4 client.
-func newGoogleSafeBrowsingV4(gsb *cmd.GoogleSafeBrowsingConfig) va.SafeBrowsing {
+func newGoogleSafeBrowsingV4(gsb *cmd.GoogleSafeBrowsingConfig, logger blog.Logger) va.SafeBrowsing {
 	// If there is no GSB configuration, don't create a client
 	if gsb == nil {
 		return nil
 	}
 	if err := configCheck(gsb); err != nil {
 		cmd.FailOnError(err, "unable to create new safe browsing v4 client")
 	}
+
+	// Create the DB file if it doesn't exist
+	dbFile := filepath.Join(gsb.DataDir, v4DbFilename)
+	dbFileHandle, err := os.Create(dbFile)
+	if err != nil {
+		cmd.FailOnError(err, fmt.Sprintf(
```

Comment:
Good point on the `err` check before `FailOnError`. I also agree RE: making it return an err instead of abort. I followed the style that I found in `gsb.go` but I'm happy to fix that up in this PR too. Thanks!